app/code/community/HE/TwoFactorAuth/Model/Observer.php

```diff
@@ -260,7 +260,18 @@ protected function _setPasswordData($observer)
         */
         $password = $observer->getEvent()->getPassword();
         $user->setOrigData('password', $password);
-        $user->setPassword();
+        /*
+            Add version compare to handle the different _beforeSave functionality in
+            Mage_Admin_Model_user.
+            In versions below 1.14.1.0, setPassword will assign the admin password to NULL,
+            therefore only allowing the admin user login once.
+            Checking for Enterprise occurs in the callAdminAuthenticate function
+         */
```

Comment:
This comment was only in regards to the code you are introducing. I do not want to do a mass re-formatting of this file.

https://github.com/sdinteractive/SomethingDigital-Guides/blob/master/Standards/HowToSubmitAGoodPR.md#11-avoid-reformatting

This is only likely to cause more issues in the future if / when we merge upstream back to our fork.